### DISCUSSION TITLE
alert modal in serenity demo page is placed under entity dialog

### DISCUSSION THREAD

**Discussion by minhhungit** [serenity-is/Serenity#6231] [General]
@volkanceylan @VictorTomaili 
How to reproduce:

open https://serenity.is/demo/Northwind/Supplier#edit/16
f12 open browser console, enter Q.alert('This is test')

=> can not hit button on alert model

![image](https://user-images.githubusercontent.com/2279508/150372004-4a1c3123-1c9a-4552-bde9-440e09e9070c.png)

**Comment by VictorTomaili** [comment]
Yes, we know this issue. It's related to the StartSharp and pro theme. So you don't need to worry about it.
